### PR TITLE
Hide other tas

### PR DESCRIPTION
**Pull request by davidhtien**

### CODE REVIEW

File: ap/accounts/management/commands/populate_saturday_tas.py [hunk 1]
```diff
@@ -20,7 +20,6 @@ def _populate_saturday_tas(self):
         ta_obj = User.objects.get(firstname=firstname, lastname=lastname)
         print 'added ', ta_obj
         sta_perm.user_set.add(ta_obj)
-        ta_perm.user_set.remove(ta_obj)
```
Comment (billyeh) [inline ap/accounts/management/commands/populate_saturday_tas.py]: No. https://github.com/attendanceproject/djattendance/pull/949/files#diff-d667b14d1239e0d226fc84bd7b72f3d4R33 

We need to remove them from the TA list for https://github.com/attendanceproject/djattendance/pull/949/files#diff-d667b14d1239e0d226fc84bd7b72f3d4R33 to work.

**Comment by billyeh** [comment]
Does this hide Saturday TAs for the trainee submission form?

File: ap/accounts/management/commands/populate_saturday_tas.py (hunk 1 above)
Comment (yipingbenjaminliu) [reply ap/accounts/management/commands/populate_saturday_tas.py]: lols, my bad, i read the code wrong. terrible character, thanks @billyeh

Comment (billyeh) [reply ap/accounts/management/commands/populate_saturday_tas.py]: U good haha

**Pull request merged**